### DISCUSSION TITLE
How to obtain more realistic Channel State Information (CSI) in Sionna RT?

### DISCUSSION THREAD

**Discussion by fighting-xq** [NVlabs/sionna#600] [Q&A]
I'm a novice regarding Sionna and wireless communication. Recently, I was using Sionna RT to obtain CSI with the help of 'cir_to_ofdm_channel', and I've noticed that the CSI is almost noise-free and perfect. What methods can I use to obtain more realistic CSI? I try to add some noisy with complex_normal. But it seems to be difficult to handle the var.

**Comment by jhoydis** [comment]
You can have a look at this [notebook](https://nvlabs.github.io/sionna/examples/OFDM_MIMO_Detection.html) for an in-depth comparison of channel estimation schemes.
